.github/workflows/integration.yaml

```diff
@@ -35,8 +35,6 @@ jobs:
       run: |
         kubectl get pods
     - uses: actions/checkout@v2
-      with:
```

Comment:
Please restore this file from the current dev branch, e.g with `git checkout origin/dev -- .github/workflows/integration.yaml `.

Reply:
will do.

Reply:
In that case I'd suggest not marking this as resolved until you've pushed the change, that makes it easier to keep an overview for reviewer and author of what is left to do.

Reply:
Sure